appium-dotnet-driver/Appium/AppiumCommand.cs

```diff
@@ -58,8 +58,8 @@ public class AppiumCommand
                 new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.StartActivity, "/session/{sessionId}/appium/device/start_activity"),
                 new AppiumCommand(CommandInfo.GetCommand,  AppiumDriverCommand.GetSettings, "/session/{sessionId}/appium/settings"),
                 new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.UpdateSettings, "/session/{sessionId}/appium/settings"),
-		new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.TouchID, "/session/{sessionId}/appium/simulator/touch_id"),
-		new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.ToggleTouchIdEnrollment, "/session/{sessionId}/appium/simulator/toggle_touch_id_enrollment"),
+		        new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.TouchID, "/session/{sessionId}/appium/simulator/touch_id"),
```

Comment:
Oh my email wasn't verified by Github when I made the commit. I have signed the CLA.
The spacing is odd, looks fine in visual studio. Maybe a spaces vs tabs issue?